### DISCUSSION TITLE
How is the "most popular" page populated and can you tweak it?

### DISCUSSION THREAD

**Discussion by ViciousBadger** [FreeTubeApp/FreeTube#2103] [Q&A] [answered]
For me it shows mostly a bunch of tech related videos but I can't figure out why. Is it related to my profile subscriptions?

**Comment by PrestonN** [comment, marked as answer]
The Most Popular tab is dependent on which Invidious instance you are connected to.

It is the latest video from the top 40 most subscribed channels from that instance. This does mean that the list of videos on this page will typically be from the same channels. It also means that changing Invidious instances will show a completely different list of videos.

Hope that helps.
